Iterable/iterable-android-sdk
# Implement Anonymous user tracking

## Description

- **Release Notes**
> We have successfully implemented functionality for tracking Anonymous User events.

> Create a known user with UUID if any criteria match and sync all the events over the iterable dashboard.

> We've created a document for Anonymous User Event Tracking.

> Develop a unit test case to verify the logic for matching criteria.

- **Test Steps**
> Trigger any event without logging into the SDK. If the criteria match your event, then it will create a user with your UUID. Using that UUID, you can search users over the iterable dashboard. Here you can see the event list in history.
